src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MemberAnalysisResult.cs

```diff
@@ -17,7 +18,16 @@ namespace Microsoft.CodeAnalysis.CSharp
     {
         // put these first for better packing
         public readonly ImmutableArray<Conversion> ConversionsOpt;
-        public readonly ImmutableArray<int> BadArgumentsOpt;
+
+        /// <summary>
+        /// A bit vector representing whose true bits indicate indices of bad arguments
+        /// </summary>
+        /// <remarks>
+        /// The capacity of this BitVector might not match the parameter count of the method overload being resolved.
+        /// For example, if a method overload has 5 parameters and the second parameter is the only bad parameter, then this
+        /// BitVector could end up with Capacity being 2 where BadArguments[0] is false and BadArguments[1] is true.
+        /// </remarks>
+        public readonly BitVector BadArguments;
```

Comment:
>BadArguments [](http://example.com/codeflow?start=34&length=12)

I am not sure why the member and the parameter got renamed. It looks like they might hold default value.

Reply:
I thought the "Opt" suffix is an old convention that is no longer used, so renamed it as I'm already changing these lines.

Should I revert the rename?

Reply:
> I thought the "Opt" suffix is an old convention that is no longer used

As far as I know, this isn't the case. The suffix is not required, but also is no forbidden.  

> Should I revert the rename?

I think this will be the right thing to do. It sounds like there a no good reasons for the rename.